x/rollup/types/params.go

```diff
@@ -0,0 +1,50 @@
+package types
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// Default parameter values
+const (
+	DefaultL1FeeRecipient string = "0x000000000000000000000000000000000000dEaD"
+	// DefaultL1CrossDomainMessenger uses the devnet address of the L1 cross domain messenger contract as the default value.
+	DefaultL1CrossDomainMessenger string = "0x9A9f2CCfdE556A7E9Ff0848998Aa4a0CFD8863AE"
+	DefaultMinFeeWithdrawalAmount uint64 = 400_000
+	DefaultFeeWithdrawalGasLimit  uint64 = 400_000
```

Comment:
I'd prefer to delete these global vars, and only have one `DefaultParams()` function that returns a `Params` object.